test/utils/nrosched/nrosched.go

```diff
@@ -119,3 +120,25 @@ func CheckNROSchedulerAvailable(cli client.Client, nroSchedName string) *nropv1a
 func IsSchedulingErrorCannotAlign(msg string) bool {
 	return strings.Contains(msg, ErrorCannotAlignPod)
 }
+
+func GetNROSchedulerName(cli client.Client) string {
```

Comment:
I think we have a duplicate of this code, don't we?

Reply:
Hmm, I might have missed that. Let me look in the code again.